_tags

```diff
@@ -1,5 +1,5 @@
 true : bin_annot, safe_string
-not <ci/**>: warn_error(+1..49), warn(A-3-4-41-44)
+not <ci/**>: warn_error(+1..49-3), warn(A-4-41-44)
```

Comment:
We could maybe add an ocamlbuild rule (this is what 0install does):

`if ci_build then flag ["compile"; "ocaml"] (S [A"-warn-error"; A"..."]);`

Would be nice to automate it somehow for all projects.